community/kernel/src/test/java/org/neo4j/kernel/impl/index/schema/SpatialLayoutTestUtil.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2002-2018 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.impl.index.schema;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+
+import org.neo4j.gis.spatial.index.Envelope;
+import org.neo4j.gis.spatial.index.curves.HilbertSpaceFillingCurve2D;
+import org.neo4j.gis.spatial.index.curves.SpaceFillingCurve;
+import org.neo4j.helpers.collection.PrefetchingIterator;
+import org.neo4j.index.internal.gbptree.Layout;
+import org.neo4j.internal.kernel.api.IndexQuery;
+import org.neo4j.kernel.api.index.IndexEntryUpdate;
+import org.neo4j.kernel.api.schema.index.IndexDescriptor;
+import org.neo4j.kernel.api.schema.index.IndexDescriptorFactory;
+import org.neo4j.test.rule.RandomRule;
+import org.neo4j.values.storable.CoordinateReferenceSystem;
+import org.neo4j.values.storable.PointValue;
+import org.neo4j.values.storable.Value;
+import org.neo4j.values.storable.Values;
+
+class SpatialLayoutTestUtil extends LayoutTestUtil<SpatialSchemaKey,NativeSchemaValue>
```

Comment:
This guy is not used anymore. What happened to all the tests?

Reply:
The file that it tested was removed when we rewrote SpatialKnownIndex. This was used for testing the Spatial Index Accessor which is now happening on the fusion level. So I guess the right thing is just to remove this unused test util.

Reply:
✅